### PR TITLE
#582

### PR DESCRIPTION
**Pull request by zjuasmn**
Challenge details: no need to show registrants tab when there is still zero registrants

### CODE REVIEW

File: src/shared/components/challenge-detail/Header/TabSelector/index.jsx [hunk 1]
```diff
@@ -77,11 +77,16 @@ export default function ChallengeViewSelector(props) {
           styleName={getSelectorStyle(selectedView, DETAIL_TABS.DETAILS, trackLower)}
         >DETAILS
         </a>
-        <a
-          onClick={(e) => { handleSelectorClicked(e, DETAIL_TABS.REGISTRANTS); }}
-          styleName={getSelectorStyle(selectedView, DETAIL_TABS.REGISTRANTS, trackLower)}
-        >REGISTRANTS {numRegistrants ? `(${numRegistrants})` : ''}
-        </a>
+        {
```
Comment (birdofpreyru) [inline src/shared/components/challenge-detail/Header/TabSelector/index.jsx]: For such conditional blocks we prefer to use `?:` operator, also no need to compare with zero then; please update to:
```
{
  numRegistrants ? (
    <a
      ...
    >...</a>
  ) : null
}
```

File: src/shared/components/challenge-detail/Header/TabSelector/index.jsx [hunk 2]
```diff
@@ -77,11 +77,16 @@ export default function ChallengeViewSelector(props) {
...
+          numRegistrants > 0 &&
+          <a
+            onClick={(e) => {
+              handleSelectorClicked(e, DETAIL_TABS.REGISTRANTS);
+            }}
+            styleName={getSelectorStyle(selectedView, DETAIL_TABS.REGISTRANTS, trackLower)}
+          >REGISTRANTS {numRegistrants ? `(${numRegistrants})` : ''}
```
Comment (birdofpreyru) [inline src/shared/components/challenge-detail/Header/TabSelector/index.jsx]: As the entire block is rendered conditionally now, no need for `{numRegistrants ? `(${numRegistrants})` : ''}`, should be just `({numRegistrants})`

**Comment by zjuasmn** [comment]
@birdofpreyru PR update.

File: src/shared/components/challenge-detail/Header/TabSelector/index.jsx [hunk 3]
```diff
@@ -78,14 +78,15 @@ export default function ChallengeViewSelector(props) {
...
-            styleName={getSelectorStyle(selectedView, DETAIL_TABS.REGISTRANTS, trackLower)}
-          >REGISTRANTS {numRegistrants ? `(${numRegistrants})` : ''}
-          </a>
+          numRegistrants ? (
+            <a
+              onClick={(e) => {
+                handleSelectorClicked(e, DETAIL_TABS.REGISTRANTS);
+              }}
+              styleName={getSelectorStyle(selectedView, DETAIL_TABS.REGISTRANTS, trackLower)}
+            >REGISTRANTS {`(${numRegistrants})`}
```
Comment (birdofpreyru) [inline src/shared/components/challenge-detail/Header/TabSelector/index.jsx]:
```
>REGISTRANTS {`(${numRegistrants})`}
```
should be just
```
>REGISTRANTS ({numRegistrants})
```
well, anyway, I'll correct myself and merge in

**Pull request merged**